integration/src/ethereum/ethereum.ts

```diff
@@ -203,38 +203,6 @@ export function ethereumTests(get: () => { wallet: core.HDWallet; info: core.HDW
       TIMEOUT
     );
 
-    test(
```

Comment:
I agree with the `ETHSignTx` type change below, but we should have separate tests for both tx formats (EIP-1559 and legacy fee parameters)

Reply:
we've got them in the form of "ethSignTx() - ETH EIP-1559" above and "ethSignTx() - ERC20" below; this test was specifically targeting providing the combination of both types of parameters, which is now illegal